controller/src/main/java/io/pravega/controller/store/stream/Stream.java

```diff
@@ -522,6 +522,14 @@
      */
     CompletableFuture<Map<UUID, ActiveTxnRecord>> getActiveTxns(OperationContext context);
 
+    /**
+     * Api to List transaction in completed(COMMITTED/ABORTED) state.
```

Comment:
Docs about semantics of batch etc should be spelled out here.

Reply:
done